cg/meta/report/report_api.py

```diff
@@ -108,14 +108,14 @@ def render_delivery_report(self, report_data: dict) -> str:
         template: Template = env.get_template(self.get_template_name())
         return template.render(**report_data)
 
-    def get_cases_without_delivery_report(self, pipeline: Pipeline) -> list[Family]:
+    def get_cases_without_delivery_report(self, pipeline: Pipeline) -> list[Case]:
         """Returns a list of cases that has been stored and need a delivery report."""
-        stored_cases: list[Family] = []
+        stored_cases: list[Case] = []
         analyses: Query = self.status_db.analyses_to_delivery_report(pipeline=pipeline)[
             :MAX_ITEMS_TO_RETRIEVE
         ]
         for analysis_obj in analyses:
-            case: Family = analysis_obj.family
+            case: Case = analysis_obj.family
```

Comment:
```suggestion
            case: Case = analysis_obj.case
```
I think we need to patch this accross all store models

Reply:
Yeah, I'll save the renaming for those fields for a separate PR.